start-site/src/main/java/io/spring/start/site/extension/dependency/springmodulith/SpringModulithBuildCustomizer.java

```diff
@@ -0,0 +1,101 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.spring.start.site.extension.dependency.springmodulith;
+
+import io.spring.initializr.generator.buildsystem.Build;
+import io.spring.initializr.generator.buildsystem.Dependency;
+import io.spring.initializr.generator.buildsystem.Dependency.Builder;
+import io.spring.initializr.generator.buildsystem.DependencyContainer;
+import io.spring.initializr.generator.buildsystem.DependencyScope;
+import io.spring.initializr.generator.spring.build.BuildCustomizer;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Registers Spring Modulith dependencies to the build file of the project to be created, in particular registering
+ * integration dependencies depending on others registered for inclusion as well (observability and persistence).
+ *
+ * @author Oliver Drotbohm
+ */
+class SpringModulithBuildCustomizer implements BuildCustomizer<Build> {
+
+	static final String GROUP_ID = "org.springframework.experimental";
+	static final String BOM_ARTIFACT_ID = "spring-modulith-bom";
+
+	static final Collection<String> OBSERVABILITY_DEPENDENCIES = List.of("datadog", "graphite", "influx", "new-relic",
+			"prometheus", "wavefront", "zipkin");
+
+	private static final String ARTIFACT_PREFIX = "spring-modulith-";
+
+	private static final Builder<?> ACTUATOR = Dependency.withCoordinates(GROUP_ID, ARTIFACT_PREFIX + "actuator")
+			.scope(DependencyScope.RUNTIME);
+
+	private static final Builder<?> OBSERVABILITY = Dependency
+			.withCoordinates(GROUP_ID, ARTIFACT_PREFIX + "observability")
+			.scope(DependencyScope.RUNTIME);
+
+	private static final Builder<?> STARTER_JDBC = Dependency.withCoordinates(GROUP_ID,
+			ARTIFACT_PREFIX + "starter-jdbc");
+
+	private static final Builder<?> STARTER_JPA = Dependency.withCoordinates(GROUP_ID, ARTIFACT_PREFIX + "starter-jpa");
+
+	private static final Builder<?> STARTER_MONGODB = Dependency.withCoordinates(GROUP_ID,
+			ARTIFACT_PREFIX + "starter-mongodb");
+
+	private static final Builder<?> TEST = Dependency.withCoordinates(GROUP_ID, ARTIFACT_PREFIX + "starter-test")
+			.scope(DependencyScope.TEST_COMPILE);
+
+	@Override
+	public void customize(Build build) {
+		DependencyContainer dependencies = build.dependencies();
+
+		// Actuator
+
+		if (dependencies.has("actuator")) {
+			dependencies.add("modulith-actuator", ACTUATOR);
+		}
+
+		// Observability
+
+		if (OBSERVABILITY_DEPENDENCIES.stream().anyMatch(dependencies::has)) {
+			dependencies.add("modulith-observability", OBSERVABILITY);
+		}
+
+		// Event publication registry support
+
+		addEventPublicationRegistryBackend(build);
+
+		dependencies.add("modulith-starter-test", TEST);
+	}
+
+	private void addEventPublicationRegistryBackend(Build build) {
+
+		var dependencies = build.dependencies();
```

Comment:
There was a previous review about the use of `var`. Can you please review the  code to remove any remaining use?